core-services/prow/02_config/_config.yaml

```diff
@@ -825,8 +844,6 @@ tide:
     - openshift-priv/external-storage
     - openshift-priv/federation-v2-operator
     - openshift-priv/file-integrity-operator
-    - openshift-priv/gcp-pd-csi-driver
```

Comment:
these non-console related changes make me wonder if something isn't quite right yet.  :thinking:

Reply:
The queries are just reordered (and they are similar so they switching places manifest as these small changes).